### ISSUE TITLE
Camera Rotation and Zoom with Mouse Gets Stuck

### ISSUE THREAD

**Issue by JustThinus** [MaFi-Games/Captain-of-Industry-issues#753]
### Issue description

When playing the game after about 10 minutes, the camera would get stuck. My scroll wheel also stops working meaning I cannot Zoom or rotate the camera with my mouse. The keyboard camera controls do work. I can still pan the map with right mouse button and everything responds to left click. I also tried alt and left mouse button to rotate the camera but it will not accept that input from my mouse.

### Expected behavior

Mouse scroll wheel and middle mouse button should keep on working and rotate and zoom the camera,

### Game version

v0.4.10

**Comment by screaney** [comment]
+1 this happens to me 100% of the time while playing, usually within the first few minutes.  Mouse wheel no longer recognized, nor middle mouse for free look.  The cursor icon changes, so the mouse IS working, however the camera doesn't respond.  Nothing in log files when it happens.  Restarting the game 'fixes' the issue, but usually happens again within a few minutes of play time.

**Comment by krasje** [comment]
So glad I'm not the only one who has this problem. I switched mouses still occurs.

**Comment by dengliziwow** [comment]
I have the same trouble too！ and even more， when the camera stuck， saving game will crash to the desktop。

**Comment by krasje** [comment]
Perhaps whit an upcoming patch it solves itself.  Until the mean time i can't play :-(

**Comment by JustThinus** [comment]
Hi Everyone, I have resolved this issue with Captain of Industry, I will be giving some detail below, maybe something applies to you guys but it is quite drastic.

I was running the insider preview of Windows 11 on my PC and my issues started after one of the many, many, many Windows updates you receive through the insider program. I wanted to get off the insider preview but to do that I had to do a clean install of Windows on my PC. After doing just that, and installing all the drivers, I installed steam and Captain of Industry and everything worked as it should.

I hope this helps.

**Comment by screaney** [comment]
> Hi Everyone, I have resolved this issue with Captain of Industry, I will be giving some detail below, maybe something applies to you guys but it is quite drastic.
> 
> I was running the insider preview of Windows 11 on my PC and my issues started after one of the many, many, many Windows updates you receive through the insider program. I wanted to get off the insider preview but to do that I had to do a clean install of Windows on my PC. After doing just that, and installing all the drivers, I installed steam and Captain of Industry and everything worked as it should.
> 
> I hope this helps.

I am also on the insider version of Win11.  I'll give this a try and see if it resolves the issue.  It does make sense, the game updates didn't coincide with when the issue started, but a Win11 insider update did.

**Comment by krasje** [comment]
I have to agree with @screaney , beside removing win 11 ad reinstalling everything is not something that i look forward too. 
If i saw it correctly in the last update they added hotkeys for zooming in/out.  Perhaps there is an issue.

**Comment by krasje** [comment]
> > Hi Everyone, I have resolved this issue with Captain of Industry, I will be giving some detail below, maybe something applies to you guys but it is quite drastic.
> > I was running the insider preview of Windows 11 on my PC and my issues started after one of the many, many, many Windows updates you receive through the insider program. I wanted to get off the insider preview but to do that I had to do a clean install of Windows on my PC. After doing just that, and installing all the drivers, I installed steam and Captain of Industry and everything worked as it should.
> > I hope this helps.
> 
> I am also on the insider version of Win11. I'll give this a try and see if it resolves the issue. It does make sense, the game updates didn't coincide with when the issue started, but a Win11 insider update did.


Wouldn't be removing the latest win 11 updates be faster than reinstalling the entire win 11?

**Comment by JustThinus** [comment]
I agree with @krasje removing the updates would be a faster option. Maybe someone can try that and give feedback if that resolved the issue. My solution is very drastic but for me it was time for a clean install anyway (I had so much nonsense installed...)

**Comment by screaney** [comment]
> I agree with @krasje removing the updates would be a faster option. Maybe someone can try that and give feedback if that resolved the issue. My solution is very drastic but for me it was time for a clean install anyway (I had so much nonsense installed...)

Good idea. I was already planning on reinstalling, but I'll try going back first and see if that resolves it.  It looks like Windows 11 Insider Preview 25188.1000 (rs_prerelease) coincides with when this issue started.

**Comment by screaney** [comment]
Since 25188 was two versions ago, I was only able to go back one, after that, the option to go back is disabled.  The issue persisted after going back just one version, but figured that would be the case.  Will try reinstalling clean now.

**Comment by screaney** [comment]
> Since 25188 was two versions ago, I was only able to go back one, after that, the option to go back is disabled. The issue persisted after going back just one version, but figured that would be the case. Will try reinstalling clean now.

Sure enough, reinstalling Win11 without the insider build resolved the issue!

**Comment by krasje** [comment]
> > Since 25188 was two versions ago, I was only able to go back one, after that, the option to go back is disabled. The issue persisted after going back just one version, but figured that would be the case. Will try reinstalling clean now.
> 
> Sure enough, reinstalling Win11 without the insider build resolved the issue!

Good news though can you prevent win 11 from reinstalling that update? Or do you have to disable all win 11 updates? If it is the last then imo this is a temporary solution :(

**Comment by JustThinus** [comment]
@krasje I agree that this is only a temporary solution, I can see three possible outcomes for this.

1. Microsoft fixes the update before the official release
2. The devs find the problem and updates the game
3. An the final outcome is allot of very unhappy gamers 

I hope that Microsoft fixes their broken code, but we'll see.

**Comment by grischpel** [comment]
I have installed Win11 Version 22H2 Build 25197.1000 
I also have this issue with the game. On a fresh start I can use the wheel and free camera. But as soon as I click on any object, the functions doesnt work anymore.

**Comment by MaFi-Games** [comment]
Closing as this seemed to be a Windows issue. Please file a new ticket if it still persists.
